dev_environment/cfg/l3afd_win.cfg

```diff
@@ -0,0 +1,56 @@
+[DEFAULT]
+
+[l3afd]
+pid-file: C:/var/l3afd/l3afd.pid
+datacenter: dc
+bpf-dir: C:/var/l3afd/repo
+bpf-log-dir: C:/var/l3afd/repo
+shutdown-timeout: 1s
+http-client-timeout: 10s
+max-ebpf-restart-count: 3
+bpf-chaining-enabled: false
+swagger-api-enabled: true
+# PROD | DEV
+environment: DEV
+# BpfMapDefaultPath is base path for storing maps
+BpfMapDefaultPath:  C:/sys/fs/bpf
```

Comment:
Pin paths on Windows are (at least currently) not filesystem paths, so no need for the C: prefix.

Reply:
okay